fe/fe-core/src/main/java/org/apache/doris/load/loadv2/SparkLoadJob.java

```diff
@@ -701,6 +702,20 @@ protected long getEtlStartTimestamp() {
         return etlStartTimestamp;
     }
 
+    public SparkLoadAppHandle getHandle() {
+        return sparkLoadAppHandle;
+    }
+
+    public void clearSparkLauncherLog() {
+        String logPath = sparkLoadAppHandle.getLogPath();
```

Comment:
Are you sure the `sparkLoadAppHandle` is not null here?

Reply:
Not sure. So I made a new check here.